images/tenacity.svg

```diff
@@ -5,7 +5,7 @@
    height="128"
    width="128"
    version="1.0"
-   sodipodi:docname="saucedacity.svg"
+   sodipodi:docname="tenacity.svg"
```

Comment:
The icon's content doesn't seem to have changed. Careful.

Reply:
I missed that one. I'll see what's in `tenacityteam/tenacity-legacy` and copy that or something.